docs/develop/app-structure.md

```diff
@@ -4,17 +4,25 @@ subtitle: Structuring your Encore application
 ---
 
 Encore uses a monorepo design and it's best to use one Encore app for your entire backend application.
-This lets Encore build an application model that spans your entire app, necessary to get the most value out of many features like [distributed tracing](/docs/observability/tracing) and [Encore Flow](/docs/develop/encore-flow).
+This lets Encore build an application model that spans your entire app, necessary to get the most value out of many
+features like [distributed tracing](/docs/observability/tracing) and [Encore Flow](/docs/develop/encore-flow).
 
 If you have a large application, see advice on how to [structure an app with several systems](/docs/develop/app-structure#large-applications-with-several-systems). 
 
-It's easy to integrate Encore applications with any pre-existing systems you might have. To make this frictionless, Encore comes with built-in tools like [client generation](/docs/develop/client-generation).
+It's easy to integrate Encore applications with any pre-existing systems you might have. To make this frictionless,
+Encore comes with built-in tools like [client generation](/docs/develop/client-generation).
 
 ## Monoliths and small microservices applications
 
-Simple architectures like monoliths, or apps with a small number of services, are straightforward to structure.
+Small architectures like monoliths, or apps with a small number of services, can be structure as a list of Encore
+services. Services are the base building blocks of your application, they separate your application in isolated units
+with their own endpoints and database (if using one). In a microservices application, an Encore service represents a
+single microservice. While services are isolated and each have their own databases by default,
+[databases may be shared between services](/docs/how-to/share-db-between-services).
 
-Simply create a Go package for each service (also known as service packages), [define APIs](/docs/develop/services-and-apis) and business logic in Go files, and add database migrations to define the structure of any database(s). (See more about [SQL databases](/docs/develop/databases).)
```

Comment:
I merged the text from main here and remove any words like "simply" in this tutorial since it's usually better to avoid any word that imply how easy things should be for users in documentation.

Reply:
Great point!